.github/renovate.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["github>hmcts/.github//renovate/global"],
+  "extends": ["local>hmcts/.github:renovate-config"],
   "labels": ["Renovate-dependencies"],
```

Comment:
The .dependabot config should probably be removed if renovate is already enabled, not really sure how we ended up with both running at the same time